### PR TITLE
Adapt short codes

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Comment by BeateRixen** [comment]
I have kept the short code "current-version" for now (and set it to 10.18) as we have a couple of occurrences under discussion at the moment and I don´t want to break the build.

**Comment by BeateRixen** [comment]
- Removed the "current-version" short code from all API links. 
- Removed several short codes no longer in use.
- Rephrased several sections towards CD.

**Review by kailol-sag — APPROVED** [review]
I only reviewed the Streaming Analytics stuff, and this looks ok to me. Thanks :-)

**Review by Oezge-Akin — APPROVED** [review]

File: themes/c8ydocs/layouts/shortcodes/c8y-current-version.html [hunk 1]
```diff
@@ -1 +1 @@
-{{- "10.19.0" -}}
+{{- "10.18.0" -}}
```
Comment (rssag) [inline themes/c8ydocs/layouts/shortcodes/c8y-current-version.html:1]: so we do keep the shortcode? :)

**Review by rssag — APPROVED** [review]

File: themes/c8ydocs/layouts/shortcodes/c8y-current-version.html (hunk 1 above)
Comment (BeateRixen) [reply themes/c8ydocs/layouts/shortcodes/c8y-current-version.html:1]: Yes, as mentioned above in my comment I keep it for now (and set it to 10.18) as we have a couple of occurrences under discussion at the moment and I don´t want to break the build.

**Pull request merged**